realm/realm-library/build.gradle

```diff
@@ -92,6 +92,9 @@ android {
     packagingOptions {
         exclude 'META-INF/NOTICE.txt'
         exclude 'META-INF/LICENSE.txt'
+        // We did strip with cmake for release build.
+        // Please, Gradle, you are not that smart! Pleas DO NOT strip debug symbols for debug build!
+        doNotStrip "*/*/*.so"
```

Comment:
Oh, so the problem is that Gradle was trying to strip binaries that were already stripped and failing to handle the case where they were already stripped?